pyproject.toml

```diff
@@ -2,8 +2,12 @@
 name = "beamer"
```

Comment:
Should we call this `beamer-agent`?

Reply:
> Should we call this `beamer-agent`?

Good question. There's already https://pypi.org/project/beamer/ which also makes it tough to just leave this as "beamer".
Let's see what we provide as part of this package:

- the `beamer-agent` executable
- the relayer executable (albeit in an "implementation-detail" manner)
- the `beamer` Python package

In the future there might be more things in there.
Considering all of this, I would go with `beamer-bridge`.
(Also, it's fine if the name of our PyPI package differs from the Python package we ship. See https://pypi.org/search/?q=bridge&o=.)

Reply:
I like ` beamer-bridge`! 👍